.env

```
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_SUPABASE_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InJsbnBveXJhcGN6cnNnbXh0bHJyIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NTk0NTc0NTYsImV4cCI6MjA3NTAzMzQ1Nn0.g22ICKPnBEe0KC7_FFDM5Dm2dV_KjadQXZIxjMl-JNI
+NEXT_PUBLIC_SUPABASE_URL=https://rlnpoyrapczrsgmxtlrr.supabase.co
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)